back/engines/commercial/public_api/spec/acceptance/v2/projects_spec.rb

```diff
@@ -135,6 +144,25 @@
       end
     end
 
+    context "when filtering by 'area_ids'" do
+      let(:areas) { create_list(:area, 2) }
+      let(:area_ids) { areas.pluck(:id) }
+
+      let!(:project) do
+        create(:project).tap { |project| project.areas << areas }
+      end
+
+      let!(:project_with_one_area) do
+        create(:project).tap { |project| project.areas << areas.first }
+      end
```

Comment:
```suggestion
      let!(:project) { create(:project, areas: areas) }

      before do
        # This project shouldn't be returned since it only has one of the requested areas.
        create(:project, areas: areas.take(1))
      end
```